frontend/next.config.js

```diff
@@ -43,4 +43,16 @@ module.exports = {
 
     return config;
   },
+  async rewrites() {
+    if (process.env.PROXY_BACKEND !== 'true') return [];
+
+    const backend = process.env.NEXT_PUBLIC_BACKEND_DOMAIN;
+
+    return [
+      {
+        source: '/backend/:path*',
```

Comment:
evil hardcoded value that should be a constant
also: evil redirect that the FE should not know nor care about